index.js

```diff
@@ -74,6 +78,8 @@ function togpx( geojson, options ) {
   else
     features = [{type:"Feature", properties: {}, geometry: geojson}];
   features.forEach(function mapFeature(f) {
+    if (!f.hasOwnProperty('properties'))
+      f.properties = {};
```

Comment:
Was there any particular reason for this? I'm not a particular fan of altering the input data here, even if it's preventing a crash for invalid input data (as per [spec](http://geojson.org/geojson-spec.html#feature-objects), a valid geojson object must have a properties object already).